### PR TITLE
New article

### PR DESCRIPTION
**Pull request by andrewBrunette**
Billing process description.

### CODE REVIEW

File: Billing/billing-process-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+### Audience
+Ecosystem and sales team members who are working with Ecosystem partner companies that wish to bill for their products in the Ecosystem Marketplace.
+
+| Step Name | Description | Responsible Team(s) |
+| --- | --- | --- |
+| Provider Signup | Provider signs up as a customer, and agrees to Supplemental Terms which establishes them legally as a Provider Partner. | Ecosystem |
+| Provider Integration | Provider completes an integration | Ecosystem |
+| Billing Model Definition | Provider Partner and ecosystem team member define the product and pricing model functionally. Select products, configurations, define price level and billing trigger for each item.  This is a design task, defining the billing model to be implemented in subsequent steps. This pricing is defined at net levels to provider | Ecosystem |
+| SKU Setup | Setting up the SKU in Platform.  This will be done by creating a ticket to Mary Cadera/Jared Ruckle (need to define these people functionally, rather than by name).  Cadera/Ruckle will create the SKU so the product can be billed. | Ecosystem to create ticket, Platform to instantiate SKUs. |
```
Comment (chrislittle) [inline Billing/billing-process-guidelines.md]: is this really ready?  why are we naming people in a kb specifically ?  seems like this isn't complete

File: Billing/billing-process-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{{{
+  "title": "Ecosystem Partner Billing Setup Process",
```
Comment (chrislittle) [inline Billing/billing-process-guidelines.md]: does this really belong in a public forum?  this seems like a bad place for it as I don't see how this helps any customer of the platform.  it seems to be a guide for internal processes in sales/ecosystem.  we have places for this like cloudhelp.ctl.io .  just want to ask as it reads pretty out there as a customer looking in the kb.  even the fact we ignore bad debt being stated public seems off.

**Comment by sdwhite** [comment]
I agree with Chris, this needs to be reviewed and the proper location for publishing determined, it does not look like it's targeted to our customers, so should not be in the public KB system.

**Comment by osbornm** [comment]
Closed per @sdwhite

**Pull request closed without merging**